kafka-connect-s3/src/main/java/io/confluent/connect/s3/storage/S3OutputStream.java

```diff
@@ -280,6 +284,14 @@ public void abort() {
     }
   }
 
+  public OutputStream wrappedForCompression() {
```

Comment:
nit: and since this looks great overall and we have some room to be our perfectionist selves, let me insist slightly on `wrapForCompression` because I prefer actions and verbs for methods of this kind that do something and return the result of their action.